spec/services/waste_exemptions_engine/registration_completion_service_spec.rb

```diff
@@ -57,10 +57,10 @@ module WasteExemptionsEngine
           expect(NewRegistration.where(reference: new_registration.reference).count).to eq(0)
         end
 
-        it "sends a confirmation email" do
+        it "sends a confirmation email to both the applicant and the contant emails" do
```

Comment:
This is because now the applicant and contact emails are the same hence 2 emails get sent. I'm not sure if it is relevant / useful to create an extra scenario to test that when the emails are the same only one is sent over. Open to suggestions :D

Reply:
I think we actually should add something. We are using the tests to both check and document behaviour so we should add something that covers the fact that the app only sends 1 email if both applicant and contact email are the same, but 2 if they are different.

However I'm fine that this can be done later in a separate PR.

Reply:
I'm happy to add it here, is related code after all :)